src/components/NavBar/NavBar.jsx

```diff
@@ -0,0 +1,169 @@
+// React Imports
+import React from 'react';
```

Comment:
See feedback for useState on Line 30 to 32 for explanation, but we should change Line 2 to:

`import React, { useState } from 'react'`;